packages/edit-site/src/components/block-editor/index.js

```diff
@@ -41,9 +63,8 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 		templateType
 	);
 	const { setPage } = useDispatch( 'core/edit-site' );
-	const ref = useRef();
 
-	useBlockSelectionClearer( ref );
+	const resizedCanvasStyles = useResizeCanvas( deviceType, true );
```

Comment:
Why is this extra hook needed, it seems iframe specific right? Can't we embed it in the iframe component?

Reply:
It was already here. It's to resize the canvas for preview. Imo it's an implementation detail.